rubin_sim/maf/mafContrib/TDEsPopMetric.py

```diff
@@ -225,7 +227,6 @@ def __init__(
         )
 
     def _some_color_pnum_detect(self, dataSlice, slicePoint, mags, t):
-        result = 1
```

Comment:
Don't know what dropping this line does, of if it has anything to do with the start mjd.

Reply:
It was defined and never used, so I just sniped it on the way by.